eventsources/sources/resource/start.go

```diff
@@ -272,17 +247,56 @@ func passFilters(event *InformerEvent, filter *v1alpha1.ResourceFilter, startTim
 	}
 	uObj := event.Obj.(*unstructured.Unstructured)
 	if len(filter.Prefix) > 0 && !strings.HasPrefix(uObj.GetName(), filter.Prefix) {
-		log.Infof("resource name does not match prefix. resource-name: %s, prefix: %s\n", uObj.GetName(), filter.Prefix)
+		log.Debugf("resource name does not match prefix. resource-name: %s, prefix: %s\n", uObj.GetName(), filter.Prefix)
```

Comment:
Don't we want to log messages as to why event failed?

Reply:
Changed back to Infof.